packages/ui-upgrade-scripts/lib/utils/execute-codemod.js

```diff
@@ -32,20 +32,20 @@ module.exports = ({
   ignore = [],
   parser = 'babylon',
   parserConfig,
-  isMetaComponentPackageMigration = false
+  isMetaComponentPackageMigration = false,
+  printToStdOut = false,
+  dryRun = false
 } = {}) => {
   info(`Running ${codemodPath}...`)
   info(`Source path: ${sourcePath}`)
-  info(`Config path: ${configPath}`)
+  if (configPath) info(`Config path: ${configPath}`)
```

Comment:
The content of this if block can be moved under line 44.

Reply:
Yeah, true, but maybe keeping it here helps the readability of the code ( all info is in one place)